store/src/java/com/zimbra/cs/imap/ImapHandler.java

```diff
@@ -1462,11 +1529,12 @@ private ImapCredentials startSession(Account account, EnabledHack hack, String t
         }
 
         setCredentials(new ImapCredentials(account, hack));
-        credentials.getImapMailboxStore().beginTrackingImap();
-        ZimbraLog.addAccountNameToContext(credentials.getUsername());
-        ZimbraLog.imap.info("user %s authenticated, mechanism=%s%s",
-                credentials.getUsername(), mechanism == null ? "LOGIN" : mechanism, startedTLS ? " [TLS]" : "");
-
+        if (!account.getName().equalsIgnoreCase(LC.zimbra_ldap_user.value())) {
+            credentials.getImapMailboxStore().beginTrackingImap();
+            ZimbraLog.addAccountNameToContext(credentials.getUsername());
+            ZimbraLog.imap.info("user %s authenticated, mechanism=%s%s",
```

Comment:
I think we'd still want to log this even for case where account is zimbra_ldap_user.  Can we move this outside of the if and/or remove special case handling completely?